### PR TITLE
Fix phrasing, add spec links

### LINKED ISSUE #276: odd phrasing differences in payload conformance statements, in media type discussions
Why `with a payload conforming to the Verifiable Credential Data Model` 
but `with a payload conforming to Verifiable Presentations`?

It seems to me that these should have been phrased identically, i.e., either —

{ `with a payload conforming to the Verifiable Credential Data Model` 
  and `with a payload conforming to the Verifiable Presentation Data Model` }

— or —

{ `with a payload conforming to Verifiable Credentials` 
  and `with a payload conforming to Verifiable Presentations` }

— and that each should link to the relevant data model specification/section.

### PR DESCRIPTION
**Pull request by decentralgabe**
Fix #276

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/pull/281.html" title="Last updated on Jun 28, 2024, 2:18 AM UTC (3400a6b)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/281/4b114ea...3400a6b.html" title="Last updated on Jun 28, 2024, 2:18 AM UTC (3400a6b)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1525,7 +1526,8 @@ <h2 id="vp-ld-json-jwt"><code>application/vp-ld+jwt</code></h2>
                 <code>application/vp-ld+jwt</code> Media Type specifically for
                 identifying a <a data-cite="JWT"></a>
                 with a payload
-                conforming to Verifiable Presentations.
+                conforming to the <a data-cite="VC-DATA-MODEL-2.0#verifiable-presentations">
+                Verifiable Presentation Data Model</a>.
```
Comment (selfissued) [inline index.html:1530]: I'm not all that comfortable elevating "Verifiable Presentation Data Model" to a _thing_.  The phrase is only used once in VCDM and then it's not capitalized.

It's more wordy, but I'd happier with something like "conforming to the Verifiable Presentations definition in the Verifiable Credential Data Model.

**Review by selfissued — CHANGES_REQUESTED** [review]
Replace all instances of "Verifiable Presentations Data Model".

File: index.html (hunk 1 above)
Comment (decentralgabe) [reply index.html]: updated

**Review by selfissued — APPROVED** [review]

**Comment by decentralgabe** [comment]
Editorial; merging.

**Pull request merged**